docs/warehouse/warehouse_starter_kit.md

```diff
@@ -0,0 +1,58 @@
+(warehouse-starter-kit-page)=
+# Warehouse: Where to Begin
+[There is a large selection of data available in the warehouse.](https://console.cloud.google.com/bigquery?project=cal-itp-data-infra&ws=!1m0) Consider this page a short guide to the most commonly used tables in our work.
+
+* [Important Links](#links)
+* [Trips](#trips)
+* [Shapes](#shapes)
+* [Daily](#daily)
+* [Other](#other)
+
+## Important Links
+* [DBT Docs Cal-ITP](https://dbt-docs.calitp.org/#!/overview) - information on all the tables in the warehouse.
+* [Example notebook](https://github.com/cal-itp/data-analyses/blob/main/starter_kit/gtfs_utils_v2_examples.ipynb)
+ using functions in `shared_utils.gtfs_utils_v2` that query some of the tables below.
+
+## Trips
+On a given day:
+* [fct_daily_scheduled_trips](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_trips)
+    * Use `gtfs_utils_v2.get_trips()`.
+    * Answer how many trips is an operator scheduled to run? How many trips did a particular route make?
+
+* [fct_observed_trips](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_observed_trips)
+    * Realtime observations of trips combined with scheduled data to get a full picture of what occurred.
+    * Find a trip's start time, where it went, and which route it is associated with.
+
+## Shapes
+* [fct_daily_scheduled_shapes](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_shapes)
+    * Use `gtfs_utils_v2.get_shapes()`.
+    * Contains `point` geometry, so you can see the length and location of a route an operator can run on a given date.
+    * Find out how many trips did an operator make for a particular route.
+    * Each shape has its own `shape_id` and `array_key`.
+    * An express version and the regular version of a route are considered two different shapes.
+
+## Daily
+For a given day:
+* [fct_daily_scheduled_stops](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_stops)
+    * Use `gtfs_utils_v2.get_stops()`.
+    * Contains `point` geometry.
+    * How many stops did an operator make? Where did they stop?
+    * How many stops did a particular transit type (streetcar, rail, ferry...)?
+    * Detailed information such as how passengers embark/disembark (ex: on a stop/at a station) onto a vehicle.
+
+* [fct_daily_schedule_feeds](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_schedule_feeds)
+    * Use `gtfs_utils_v2.schedule_daily_feed_to_organization()`.
```

Comment:
I guess this is out of scope for this PR but I am a bit confused about this function -- it does not actually map to an organization AFAICT, it only maps to a dataset (which could then be mapped to an organization via `dim_provider_gtfs_data` for example)